oh.ps1

```diff
@@ -269,12 +269,81 @@ function script_menu {
 }
 
 ###################################################################
-function get_confirmation {
+function get_confirmation ($arg) {
 	$choice = Read-Host -Prompt "(y/n) ? "
```

Comment:
Nit pick:  why are there 19 occurrences in this PR where there is a space before the question mark?   That isn't typically how a question is written.

Reply:
Thank you david, fixed